http-clients/src/main/java/org/triplea/http/client/lobby/moderator/BanDurationFormatter.java

```diff
@@ -8,9 +8,11 @@
 public class BanDurationFormatter {
 
   public static String formatBanMinutes(final long banMinutes) {
-    Preconditions.checkState(banMinutes > 0);
+    Preconditions.checkState(banMinutes >= 0);
 
-    if (banMinutes < 60) {
+    if (banMinutes == 0) {
+      return "less than a minute";
```

Comment:
Probably should ignore this one, method is nothing but if/else with return statements, the +1 here is not material. Refactoring to avoid this is not likely going to help.